Homework2/Homework2/Project.cs

```diff
@@ -0,0 +1,73 @@
+namespace Homework2;
+
+public class Project
+{
+    private string _name;
+    private string _description;
+    private string _deadline;
+    private decimal _budget;
+    private List<Staff> _staff;
+    private List<Tasks> _tasks;
+
+    public string Name
+    {
+        get => _name;
+        set
+        {
+            if (int.TryParse(value, out _))
+                throw new AggregateException("Name is num");
```

Comment:
I think this should be ArgumentException

Reply:
Fixed